### DISCUSSION TITLE
Can static endpoint discovery be used with a discovery server?

### DISCUSSION THREAD

**Discussion by VladimirVassiliev** [eProsima/Fast-DDS#2923] [Q&A] [answered]
In other words, can a discovery server be used only for participant discovery?

**Comment by MiguelCompany** [comment, marked as answer]
No. But depending on your use case, you could use [static endpoint discovery](https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/static.html#static-edp) + [initial peers](https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/wifi/initial_peers.htm)
